docs/docs/consumer.mdx

```diff
@@ -85,7 +85,15 @@ async def get_with_base_voltage(service: NetworkService, client: NetworkConsumer
     if not equipment:
         return
 
-    mrids = service.get_unresolved_reference_mrids(resolver.ce_base_voltage(equipment))
-    if mrids:
-        await client.get_identified_object(service, mrids[0])
+    # Get all unresolved references
```

Comment:
Should update this example to be more sensible. People will end up pulling in too much stuff.